### ISSUE TITLE
ImageBackground does not announce "disabled"

### ISSUE THREAD

**Issue by amarlette** [facebook/react-native#30936]
## Description
When using a screen reader this component does not announce that it is disabled.

This issue is covered in-depth by parent issue #30840, please check there for more details.

## React Native version:
v0.63

**Comment by pranshuchittora** [comment]
`ImageBackground` is composed of an `Image` component wrapped by a `View`. As `View` already implements the `AccessibilityState`, therefore we just need to pass the prop from ImageBackground to the wrapped view.

Is that a correct approach?

**Comment by fabriziobertoglio1987** [comment]
This issue was fixed with my pr https://github.com/facebook/react-native/issues/30936 
I tested the master branch which includes this fix and used an `ImageBackground` instead of `Image`, disabled is announced accordingly as in the below test result.

**<details><summary>CLICK TO OPEN TESTS RESULTS</summary>**
<p>

Enable audio to hear the screenreader

TEST SCENARIO
- The user moves the screenreader focus to an ImageBackground and the screenreader reads the ImageBackground accessibilityLabel "plain network image"

RESULT
- The screenreader announces the accessibilityState disabled after reading the ImageBackground accessibilityLabel "plain network image"

```javascript
<ImageBackground
  accessible={true}
  accessibilityLabel="plain network image"
  accessibilityState={{disabled: true}}
  source={fullImage}
  style={styles.base}
/>
```

<video src="https://user-images.githubusercontent.com/24992535/117655136-42ab5680-b1c9-11eb-8302-54944c443acf.mp4" width="700" />

</p>
</details>

**Comment by amarlette** [comment]
Thank you @fabriziobertoglio1987 for your pull request to this project! I will close out this issue an mark it resolved on the parent issue.

We are one step closer to a more accessible React Native thanks to you!
